src/item.c

```diff
@@ -192,7 +192,7 @@ bool8 CheckBagHasSpace(u16 itemId, u16 count)
     if (pocket != BERRIES_POCKET)
         slotCapacity = MAX_BAG_ITEM_CAPACITY;
     else
-        slotCapacity = MAX_BERRY_CAPACITY;
+        slotCapacity = MAX_BAG_ITEM_CAPACITY;
```

Comment:
Both cases do the same.

Reply:
I think the local variable here is pointless as well, but I'll just limit myself to `slotCapacity = MAX_BAG_ITEM_CAPACITY;` without the `if`/`else` unless instructed otherwise,